### PR TITLE
Controller no-ops and Controller Reset Fixes

### PR DESCRIPTION
**Pull request by cremebrule**
add controller no-op commands and proper protocols for avoiding controller leakage between env resets

### CODE REVIEW

**Review by cgokmen — APPROVED** [review]
LGTM

File: omnigibson/controllers/ik_controller.py [hunk 1]
```diff
@@ -251,25 +278,28 @@ def _command_to_control(self, command, control_dict):
         pos_relative = np.array(control_dict["{}_pos_relative".format(self.task_name)])
         quat_relative = np.array(control_dict["{}_quat_relative".format(self.task_name)])
 
-        # The first three values of the command are always the (delta) position, convert to absolute values
-        dpos = command[:3]
-        target_pos = pos_relative + dpos
+        # The first three values of the command are always the (delta) position, convert to absolute values if needed
```
Comment (ChengshuLi) [inline omnigibson/controllers/ik_controller.py:281]: This comment is no longer correct?

File: omnigibson/controllers/joint_controller.py [hunk 2]
```diff
@@ -132,6 +128,23 @@ def is_grasping(self):
...
+                # Zero values
+                cmd = np.zeros(self.command_dim)
+            elif self._motor_type == "velocity" or self._motor_type == "effort":
+                # Run negative delta
+                cmd = -control_dict["joint_{}".format(self._motor_type)][self.dof_idx]
+            else:
+                raise ValueError(f"Got invalid motor type: {self._motor_type}!")
+        else:
+            # Directly use info from control dict
+            cmd = control_dict["joint_{}".format(self._motor_type)][self.dof_idx]
```
Comment (ChengshuLi) [inline omnigibson/controllers/joint_controller.py:144]: hmm I am confused, I thought no op wants the robot to stay still. 
For absolute control, shouldn't we do the following?
- position: cmd = current joint position
- velocity/torque: 0

**Review by ChengshuLi — APPROVED** [review]

File: omnigibson/controllers/ik_controller.py (hunk 1 above)
Comment (cremebrule) [reply omnigibson/controllers/ik_controller.py]: updated!

File: omnigibson/controllers/joint_controller.py (hunk 2 above)
Comment (cremebrule) [reply omnigibson/controllers/joint_controller.py]: oops! you're right, great catch. fixed!

**Pull request merged**